requirements.yml

```diff
@@ -1,13 +1,6 @@
 ---
 
 - src: ome.postgresql
-  version: 5.2.0
```

Comment:
Is this removal on purpose? Like in https://github.com/ome/prod-playbooks/pull/379#discussion_r1844159333, the downside not being declarative is that backwards-incompatible changes to the roles might cause the playbook execution to break in the future.

As this is an example repository, this is certainly less critical than in the case of the playbooks which capture the state of production deployments.

Reply:
fixed in https://github.com/ome/ansible-example-omero-addons/pull/7/commits/6fe20367d5c11f96117374772d9d6d5a491c4b91